packages/tools/pie-icons-webc/PieIconComponent.ts

```diff
@@ -51,10 +57,19 @@ export abstract class PieIconComponent extends LitElement {
     }
 
     updateIconSize (): void {
-        const svgSize = getSvgProps(this.class, '', this.size, this.name);
+        const svgSize = getSvgProps(this.defaultClasses, '', this.size, this.name);
         this._svgWidth = svgSize.width;
         this._svgHeight = svgSize.height;
     }
 
+    attributeChangedCallback (name: string, oldVal: string | null, newVal: string | null) {
+        if (name === 'class' && newVal !== null) {
+            const incomingClasses = newVal.trim().split(' ').filter((cls) => !this.defaultClasses.split(' ').includes(cls));
+            this.class = [this.defaultClasses, ...incomingClasses].join(' ');
+        } else {
+            super.attributeChangedCallback(name, oldVal, newVal);
+        }
+    }
+
```

Comment:
Do you think we can use the `updated` callback from lit here instead? 
Mainly because the lit docs mention that `super` must be called when the native web component `attributeChangedCallback` callback is used and we are conditionally calling it? 

https://lit.dev/docs/api/LitElement/#LitElement.attributeChangedCallback

Reply:
I believe Lit adds extra logic to the native web component `attributeChangedCallback`. That’s why we need to call super, so the Lit side effects and logic are triggered correctly on top of any logic like above. However, that won’t be the case here as we are overriding the entire life cycle entirely.

I haven't tested any edge case and not aware of any but as long as we are aware of the risk and the docs, we should be fine.